assignment.md

```diff
@@ -11,63 +11,76 @@ If the version control concepts or `git` commands are still confusing to you, co
         ```
 
     - b. Who was the last person to modify `README.md`? (Hint: use `git log` with an argument)
         ```bash
-        # git command and output here
+        # git log -1 README.md
+      output: commit 7897d58f9a9334bcc55a373734c6a0c5a765d95a Author: Xihe Xie <axiezai@gmail.com>Date:   Sat Feb 6 11:21:02 2021 -0500
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
+      >"added ch2 assignment" 
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > Git stash saves changes I made locally elsewhere that's not on my repo, and it restores my repository to the last commit, any uncommitted changes I had were reversed when the file was opened again
```

Comment:
It saves the modifications you made in the repo, it's just hidden away from you with magical git tracker tricks.